<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="195266" started-at="2020-03-06T18:16:27Z" finished-at="2020-03-06T18:19:42Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="195266" started-at="2020-03-06T18:16:27Z" finished-at="2020-03-06T18:19:42Z">
      <class name="testCases.TC001">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:testCases.TC001@68c4039c]" name="beforeSuite" is-config="true" duration-ms="25609" started-at="2020-03-06T18:16:02Z" finished-at="2020-03-06T18:16:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="FAIL" signature="test(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:testCases.TC001@68c4039c]" name="test" duration-ms="186667" started-at="2020-03-06T18:16:27Z" data-provider="User" finished-at="2020-03-06T18:19:34Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[nitinmanojkumar@live.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[My@12345]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[manoj]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[TCS]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[9879876756]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[provide your creds]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[your pwd]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[manojkumarkaalimuthu@gmail.com]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[Subject to it]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
Error in configuring Email expected [true] but found [false],
Error in sending Email expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
Error in configuring Email expected [true] but found [false],
Error in sending Email expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
at testCases.TC001.test(TC001.java:54)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method status="PASS" signature="deactivate()[pri:0, instance:testCases.TC001@68c4039c]" name="deactivate" is-config="true" duration-ms="8383" started-at="2020-03-06T18:19:34Z" finished-at="2020-03-06T18:19:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deactivate -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:testCases.TC001@68c4039c]" name="afterSuite" is-config="true" duration-ms="0" started-at="2020-03-06T18:19:43Z" finished-at="2020-03-06T18:19:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- testCases.TC001 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
